Namespace migration (#300)
Vonage/vonage-java-sdk: README.md

# Vonage Server SDK for Java

> [![Maven Release](https://maven-badges.herokuapp.com/maven-central/com.nexmo/client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.nexmo/client)

Reviewer: ```suggestion
[![Maven Release](https://maven-badges.herokuapp.com/maven-central/com.vonage/client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.vonage/client)
```

Author: I can just change this here and it will work?

Reviewer: Yeah, that's why I prefer to offer suggestions like this when I can rather than a comment. If you commit the suggestions inside the GitHub UI then it'll add it to the code in this PR for you.

> [![Build Status](https://travis-ci.org/Nexmo/nexmo-java.svg?branch=version-2)](https://travis-ci.org/Nexmo/nexmo-java)

Reviewer: ```suggestion
[![Build Status](https://travis-ci.org/Vonage/vonage-java-sdk.svg?branch=version-2)](https://travis-ci.org/Vonage/vonage-java-sdk)
```

Author: I think we are getting rid of travis all together for Github Actions.

<img src="https://developer.nexmo.com/assets/images/Vonage_Nexmo.svg" height="48px" alt="Nexmo is now known as Vonage" />

> You can use this Java Server SDK to add [Vonage's API](https://developer.nexmo.com/api) to your application. To use this, you'll

Reviewer: ```suggestion
You can use this Java Server SDK to add [Vonage APIs](https://developer.nexmo.com/api) to your application. To use this, you'll
```

**Note**: We *strongly recommend* that you use a tool that supports dependency management, such as [Gradle], [Maven],
or [Ivy]

We provide a [ZIP file for each release](https://github.com/Vonage/vonage-java-sdk/releases/),

> containing the server sdk JAR, along with all the dependencies. Download the file, unzip it, and add the JAR files

Reviewer 2: server sdk -> Server SDK

Author: fixed

or [Ivy]

We provide a [ZIP file for each release](https://github.com/Vonage/vonage-java-sdk/releases/),

> containing the server sdk JAR, along with all the dependencies. Download the file, unzip it, and add the JAR files

Reviewer 2: SDK

Author: fixed

## Usage

* For help understanding our APIs, check out our awesome [developer portal](https://developer.nexmo.com/)

> * Check the [Javadoc](http://vonage.github.io/vonage-java-sdk/) for full reference documentation.

Reviewer: This redirects to https://github.com/Vonage/vonage-java-code-snippets, maybe just make it official here?

> * There are also **many useful code samples** in our [nexmo-community/nexmo-java-quickstart](https://github.com/nexmo-community/nexmo-java-quickstart) repository.

Reviewer: This redirects to https://github.com/Vonage/vonage-java-code-snippets so maybe just change it here?

## License

> This library is released under the [Apache License][license]

Reviewer 3: Apache 2.0 I think would be more correct here?

Author: fixed

> It is a good idea to [talk to us](https://nexmo-community-invite.herokuapp.com/)

Reviewer: Not sure why this is a custom heroku app link. It directs to https://developer.nexmo.com/community/slack

Author: i can change that to the redirected link instead.
